grails-app/i18n/messages_de.properties

```diff
@@ -1069,6 +1069,7 @@ frontend.de.iteratec.isr.measurand.IMAGE_TOTAL_BYTES=Bytes Images
 frontend.de.iteratec.isr.measurand.CSS_TOTAL_BYTES=Bytes CSS
 frontend.de.iteratec.isr.measurand.HTML_TOTAL_BYTES=Bytes HTML
 frontend.de.iteratec.isr.measurand.FIRST_MEANINGFUL_PAINT=First Meaningful Paint
+frontend.de.iteratec.isr.measurand.FIRST_PAINT=First Paint
```

Comment:
English label for 'frontend.de.iteratec.isr.measurand.FIRST_PAINT' is missing.

Reply:
Thanks for the catch. I added them.